packages/cfd/src/Containers/dmt5-trade-modal.tsx

```diff
@@ -95,226 +96,232 @@ const getTitle = (market_type: string, show_eu_related_content: boolean) => {
-                className='cfd-trade-modal__download-center-text'
-                size={isMobile() ? 'xxxs' : 'xxs'}
-                weight='bold'
-            >
-                {localize(
-                    'Download {{ platform }} on your phone to trade with the {{ platform }} {{ account }} account',
-                    {
-                        platform: getCFDPlatformLabel(CFD_PLATFORMS.MT5),
-                        account: getAccountTitle(),
-                    }
-                )}
-            </Text>
-            <div className='cfd-trade-modal__download-center-options'>
-                <div className='cfd-trade-modal__download-center-options--mobile-links'>
-                    <a href={getPlatformMt5DownloadLink('ios')} target='_blank' rel='noopener noreferrer'>
-                        <Icon icon='IcInstallationApple' width={135} height={40} />
-                    </a>
-                    <a href={getPlatformMt5DownloadLink('android')} target='_blank' rel='noopener noreferrer'>
-                        <Icon icon='IcInstallationGoogle' width={135} height={40} />
-                    </a>
-                    <a href={getPlatformMt5DownloadLink('huawei')} target='_blank' rel='noopener noreferrer'>
-                        <Icon icon='IcInstallationHuawei' width={135} height={40} />
-                    </a>
-                </div>
-                {!isMobile() && (
-                    <div className='cfd-trade-modal__download-center-options--qrcode'>
-                        <img src={getUrlBase('/public/images/common/mt5_download.png')} width={80} height={80} />
-                        <Text align='center' size='xxs'>
-                            {localize('Scan the QR code to download {{ platform }}.', {
-                                platform: getPlatformSettings('mt5').name,
+                        <a
+                            className='dc-btn cfd-trade-modal__download-center-app--option-link'
+                            type='button'
+                            href={getMT5WebTerminalLink({
+                                category: mt5_trade_account.account_type,
+                                loginid: (mt5_trade_account as TTradingPlatformAccounts).display_login,
+                                server_name: (mt5_trade_account as DetailsOfEachMT5Loginid)?.server_info?.environment,
                             })}
-                        </Text>
+                            target='_blank'
+                            rel='noopener noreferrer'
+                        >
+                            <Text size='xxs' weight='bold' color='prominent'>
+                                {localize('Open')}
+                            </Text>
+                        </a>
                     </div>
-                )}
+                    {!is_mobile && (
+                        <React.Fragment>
+                            <div className='cfd-trade-modal__download-center-app--option'>
```

Comment:
I suggest we make this part a separate component. It is not in the scope of our changes but since we're doing it now, let's not do double work later.